crates/ironrdp-server/src/encoder/mod.rs

```diff
@@ -203,7 +203,7 @@ impl UpdateEncoder {
                 width.get().into(),
                 height.get().into(),
                 &bitmap.data,
-                bitmap.stride,
+                bitmap.stride.into(),
```

Comment:
suggestion: I largely prefer `get` over `into` (and I’m generally averse to `into` when `from` can be used, because it’s harder to read/review)